### PR TITLE
Adds support for newer version of Leetcode and fixes few issues

### PR DESCRIPTION
**Pull request by chaudharydeepanshu**
Thanks for this chrome extension.  

I have made a few changes and fixes to it:

- Added support for the newer version of Leetcode. The extension was not working properly on the latest version of Leetcode, this change ensures that the extension works as expected on the latest version while also working on the old version.
**New Leetcode UI**
image
**Old Leetcode UI**
image
- Fixed the issue where too many new lines were being added, which was causing formatting issues.
- Fixed the issue where unwanted HTML tags were being added, which was causing incorrect formatting of the copied text.
- Updated manifest version to 3.

These changes should improve the overall functionality and usability of the extension for the users.

Please let me know if you have any questions or concerns.

### CODE REVIEW

File: source/main.js [hunk 1]
```diff
@@ -49,74 +55,121 @@ const MARKDOWN = {
...
+      document.querySelector('[data-key=description-content]').children
+    )[0].children[1];
+    text = descriptionContent.textContent.replace(/(\n){2,}/g, '\n\n').trim();
+    html = descriptionContent.innerHTML;
+    if (text == null || html == null) {
+      throw "Old version elements not found";
+    }
+  } catch (err) {
+    // If the elements for the old version are not found, try finding the elements for the new version.
+    title = document.querySelector('.mr-2.text-lg.font-medium.text-label-1.dark\\:text-dark-label-1').innerText;
```
Comment (eddgr) [inline source/main.js:83]: it looks like this targets only the element in dark mode

File: source/main.js [hunk 2]
```diff
@@ -49,74 +55,121 @@ const MARKDOWN = {
...
-      display: flex;
-    `;
+    position: absolute;
+    top: 1rem;
+    right: 0;
+    display: flex;
+  `;
+  } catch (err) {
+    // If the old version elements are not found, try finding the new version elements.
+    target = document.querySelector('.mr-2.text-lg.font-medium.text-label-1.dark\\:text-dark-label-1');
```
Comment (eddgr) [inline source/main.js:142]: same here

**Review by eddgr — CHANGES_REQUESTED** [review]
Great PR @chaudharydeepanshu! Appreciate your help with maintaining the extension.

Just had a few comments.

File: source/main.js (hunk 1 above)
Comment (chaudharydeepanshu) [reply source/main.js:83]: Thanks for checking it out. But it looks like the element names aren't actually connected to the theme color. I don't know why LeetCode named them that way. At first, I also thought they were.

**As you can see it works on light mode also:**
image

**Markdown generated for above problem in light mode:**

# [58. Length of Last Word](https://leetcode.com/problems/length-of-last-word/)

Given a string `s` consisting of words and spaces, return the length of the **last**  word in the string.

A **word**  is a maximal substring consisting of non-space characters only.

**Example 1:** 

```
Input: s = "Hello World"
Output: 5
Explanation: The last word is "World" with length 5.

```

**Example 2:** 

```
Input: s = "   fly me   to   the moon  "
Output: 4
Explanation: The last word is "moon" with length 4.

```

**Example 3:** 

```
Input: s = "luffy is still joyboy"
Output: 6
Explanation: The last word is "joyboy" with length 6.

```

**Constraints:** 

- `1 <= s.length <= 10^4`
- `s` consists of only English letters and spaces `' '`.
- There will be at least one word in `s`.

File: source/main.js (hunk 2 above)
Comment (chaudharydeepanshu) [reply source/main.js:142]: Same as above here also the element names aren't actually connected to the theme color

Comment (eddgr) [reply source/main.js:142]: Got it, my mistake. Thanks!

**Review by eddgr — APPROVED** [review]

**Pull request merged**